pkg/build/generated/internalclientset/typed/build/internalversion/build_client.go

```diff
@@ -13,7 +13,7 @@ type BuildInterface interface {
 	BuildConfigsGetter
 }
 
-// BuildClient is used to interact with features provided by the build.openshift.io group.
+// InternalBuildClient is used to interact with features provided by the build.openshift.io group.
```

Comment:
Any idea why this has changed?

Reply:
will revert... probably bad replace in path...